packages/aws-cdk-lib/aws-lambda-nodejs/lib/bundling.ts

```diff
@@ -418,10 +418,13 @@ function toTarget(runtime: Runtime): string {
 
 function toCliArgs(esbuildArgs: { [key: string]: string | boolean }): string {
   const args = new Array<string>();
+  const ls = ['--alias', '--drop', '--pure', '--log-override', '--out-extension'];
```

Comment:
what does `ls` stand for here? Should this variable have a better/clearer name? 

Also, does it make sense maybe to have this as a top level constant instead of just being a local variable here? 
(I'm not sure about this because I'm not familiar with other functionalities or the structure of this file)

Reply:
I have changed the name to `reSpecifiedKeys`, making it more specific to what the list is about. And I have kept the list inside `toCliAgs` function as it is being used inside this function only and for other functions too they have their constants declared inside the Function only.